README.md

```diff
@@ -32,6 +32,23 @@ In addition to the `admission` scenario, Gatekeeper's audit functionality allows
 
 Finally, Gatekeeper's engine is designed to be portable, allowing administrators to detect and reject non-compliant commits to an infrastructure-as-code system's source-of-truth, further strengthening compliance efforts and preventing bad state from slowing down the organization.
 
+## Admission Webhook Fail-Open Status
```

Comment:
Should we add a `Known Risk:`

Reply:
I don't want to be too alarmist. My hope is that moving this above the installation instructions will give people context. "known risk" sounds to me like "installing this opens up known vulnerabilites" rather than "enforcement at boundary not 100%", which is what I'm hoping "fail open" communicates.